openedx/features/survey_report/models.py

```diff
@@ -35,3 +35,31 @@ class SurveyReport(models.Model):
     class Meta:
         ordering = ["-created_at"]
         get_latest_by = 'created_at'
+
+
+class SurveyReportUpload(models.Model):
+    """
+    This models stores information about send request of a survey report.
+
+    .. no_pii:
+
+    fields:
+    - sent_at: Date when the report was sent.
+    - report: The report that was sent.
+    - status: Request status code.
+    - request_details: Information about the send request.
+    """
+    sent_at = models.DateTimeField(auto_now=True, help_text="Date when the report was sent.")
```

Comment:
was sent where? Maybe we should add: sent to an external URL or service. Or when was generated?

Reply:
@mariajgrimaldi this will be sent to different endpoints, maybe this will be sent to an API Rest or to a Drive Folder or to Zapier, this field is just to save the date when the report was sent.